pkgs/by-name/my/mymake/package.nix

```
@@ -0,0 +1,43 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mymake";
+  version = "2.3.6";
+
+  src = fetchFromGitHub {
+    owner = "fstromback";
+    repo = "mymake";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-UQjvxdOvD9O6TrzBFJwh3CistDGZM9HZbcwVPx1n4+A=";
+  };
+
+  buildPhase = ''
+    runHook preBuild
+
+    ./compile.sh mm
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp mm $out/bin
```

```suggestion
    install -Dm755 mm $out/bin/mm
```
Using install looks nicer, and makes sure permissions are setup correctly. It's another pattern imo should be used more than it is. @awwpotato do you have any feelings on this?

---

I agree that using `install` is better, but the nixpkgs manual seems to include both.